<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distance Decay Map</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Данные</string>
    </property>
    <addaction name="importAction"/>
    <addaction name="exportAction"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Карта</string>
    </property>
    <addaction name="reloadAction"/>
    <addaction name="increaseZoomAction"/>
    <addaction name="decreaseZoomAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuData"/>
   <addaction name="menuMap"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="quitAction"/>
   <addaction name="openAction"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="importAction"/>
   <addaction name="exportAction"/>
   <addaction name="separator"/>
   <addaction name="reloadAction"/>
   <addaction name="increaseZoomAction"/>
   <addaction name="decreaseZoomAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icon_document_open.png</normaloff>:/icons/icons/icon_document_open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icon_exit.png</normaloff>:/icons/icons/icon_exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="importAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icon_import.png</normaloff>:/icons/icons/icon_import.png</iconset>
   </property>
   <property name="text">
    <string>Импорт</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icon_export.png</normaloff>:/icons/icons/icon_export.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="reloadAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icon_refresh.png</normaloff>:/icons/icons/icon_refresh.png</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
  </action>
  <action name="increaseZoomAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icon_zoom_in.png</normaloff>:/icons/icons/icon_zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Увеличить масштаб</string>
   </property>
  </action>
  <action name="decreaseZoomAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/icon_zoom_out.png</normaloff>:/icons/icons/icon_zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Уменьшить масшаб</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
